hr_attendance_calendar/README.rst

```diff
@@ -0,0 +1,26 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Compassion CRM - Human Resources Attendance Calendar
```

Comment:
This module could be shared with OCA. Remove Compassion reference in the readme. We should as well find a better title for the module as it is used to keep track of the work hours of the employees.

Reply:
I still leave Compassion as maintainer, right?